packages/modules/imodel-browser/src/containers/iModelGrid/useIModelData.ts

```diff
@@ -104,6 +102,7 @@ export const useIModelData = ({
     const paging = `&$skip=${skip}&$top=${top}`;
     const searching = searchText?.trim() ? `&$search=${searchText}` : "";
 
+    const abortController = new AbortController();
```

Comment:
Why did you need to change this?
I think now it might work a little different as we gonna recreate it everytime any of the deps change :?